backends/arm/test/ops/test_conv.py

```diff
@@ -296,7 +299,6 @@ def _test_conv2d_u55_BI_pipeline(
             .check_count({"torch.ops.higher_order.executorch_call_delegate": 1})
             .check_not(["executorch_exir_dialects_edge__ops_aten_convolution_default"])
             .to_executorch()
-            .serialize()
```

Comment:
I guess this doesn't add much value?

Reply:
This stage produces the final .pte which I think is valuable. It also follows the pattern where there is a stage for each artifact + a run_stage method. For the serialize stage, the run_stage launches the FVP

Reply:
I meant, you are dropping serialize call for this where you are not actually running it.

Reply:
Ah right, yeah it's a "make sure it doesn't crash" test for now. Actually running is the next step!